dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CDITests.java

```diff
@@ -31,15 +30,12 @@
 @Mode(FULL)
 public class CDITests extends CommonWebServerTests {
 
-    private static final String TEST_APP_15_WITH_CDI = "testSpringBootApp15WithCDIFeatureEnabled";
     private static final String TEST_APP_20_WITH_CDI = "testSpringBootApp20WithCDIFeatureEnabled";
 
     @Override
     public String getApplication() {
         String methodName = testName.getMethodName();
-        if (TEST_APP_15_WITH_CDI.equals(methodName)) {
-            return SPRING_BOOT_15_APP_BASE;
-        } else if (TEST_APP_20_WITH_CDI.equals(methodName)) {
+        if (TEST_APP_20_WITH_CDI.equals(methodName)) {
```

Comment:
Do we need developers to check over actual FAT test changes?

Reply:
This is from commit 068c82f8a3e213c6ec3a3645d074047c6d8cad64, which Anjum from OSGi apps team did. Her team is the SME on Spring Boot tests and etc.